module_5/oop_classes.py

```diff
@@ -0,0 +1,171 @@
+from datetime import date, datetime
+import random
+
+class CommonMethods:
+    """
+    Class contains common methods
+    """
+    def __create_output_file__(self, output_file):
+        """
+        Method to create output file
+        """
+        if not output_file:
+            open(output_file, 'a').close()
+
+    def populate_output_file(self, title, section_text, city_exp_date, calculated_val, output_file='/Users/shaliyu1/Documents/python_for_dqe/module_5/PDQE_newspaper.txt'):
```

Comment:
Don't specify a special path.

For example, your code works for me like this.
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shaliyu1/Documents/python_for_dqe/module_5/newspaper.txt'

Reply:
fixed